cobalt/build/configs/BUILD.gn

```diff
@@ -55,4 +55,8 @@ config("buildflag_defines") {
   if (sb_is_modular) {
     defines += [ "ENABLE_BUILDFLAG_MODULAR_BUILD" ]
   }
+  if (cobalt_is_release_build) {
+    defines +=
+        [ "COBALT_IS_RELEASE_BUILD" ]  # This is equivalent to is_qa in Cobalt.
```

Comment:
Follow go/cobalt-naming#needed-in-upstream-code to add a buildflag instead of a define